docs/git-commands.md

```diff
@@ -0,0 +1,94 @@
+# Git Hub Command Guide
+The following is number of scenarios that you will encounter while contributing to this repo, a convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo, this clone or copy will have the remote name of `origin`.
+
+## Add your own Remote
+Once you have the origin remote you will need to add a remote to your own fork of the repo. This can be done with the following command, just replace the placeholder with your own github name.
+
+It should also be noted you will need to have forked the repo, how to do this can be seen in Tutorial.md
+
+```
+git remote add <<your-name>> https://github.com/<<Your Name Here>>/SchoolWebsite.git
+```
+
+## Commiting and Creating PR's
+So now you have the repo downloaded, you have your fork remote and the main repo remote added you can begin to work and create your own PR against the main repo. To do this, you must create a branch. It is a good convention to give the branch a descriptive name, usually something descibing the work in the PR, keep this name short and too the point, example `add-contact-page`
+
+```
+git checkout -b <<desciptive-name-here>>
+```
+Now you are working on a new branch, when you are happy with your changes or just want to save the changes to the branch you can commit it, it is good practice to run the following command first to see what files have been changed by your work.
+```
+git status
+```
+If all the files listed are the ones you are happy with to commit to your branch run the following series of commands.
+```
+git add .
+git commit -m '<<describe the changes here>>'
+git push <<your-remote-name>> <<your-branch-name>>
+```
+Now you have pushed your changes on your branch to your remote, refer to tutorial.md on how to create a PR from here
+
+## Checking out other peoples PR's
+In order to check out someones PR you need to add that persons remote, this can be done with the following command, just replace the tags with the persons github name
+```
+git remote add <<persons-github-name>> https://github.com/<<persons-github-name>>/SchoolWebsite.git
```

Comment:
could we change the first `<<persons-github-name>>` to `<<persons name>>` to match the `git remote add` from above?